build.gradle

```diff
@@ -59,19 +58,20 @@ subprojects {
 
         // They are once excluded from transitive dependencies of other dependencies,
         // and added explicitly with versions exactly the same with embulk-core:0.10.31.
-        compile "com.fasterxml.jackson.core:jackson-annotations:2.6.7"
-        compile "com.fasterxml.jackson.core:jackson-core:2.6.7"
-        compile "com.fasterxml.jackson.core:jackson-databind:2.6.7"
-        compile "com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.6.7"
-        compile "javax.validation:validation-api:1.1.0.Final"
-
-        compile "org.embulk:embulk-util-json:0.1.1"
-        compile "org.embulk:embulk-util-timestamp:0.2.1"
-        compile "org.embulk:embulk-util-retryhelper:0.8.2"
-
-        testCompile "org.embulk:embulk-junit4:0.10.31"
-        testCompile "org.embulk:embulk-core:0.10.31"
-        testCompile "org.embulk:embulk-deps:0.10.31"
+        implementation "com.fasterxml.jackson.core:jackson-annotations:2.6.7"
+        implementation "com.fasterxml.jackson.core:jackson-core:2.6.7"
+        implementation "com.fasterxml.jackson.core:jackson-databind:2.6.7"
```

Comment:
We confirmed that we can use `jackson-databind:2.6.7.5` safely here as it has totally the same set of classes with `jackson-datadbind:2.6.7`.

Reply:
Thanks fixed in [85b6772](https://github.com/embulk/embulk-output-jdbc/pull/317/commits/85b6772cc22920d44fb582315db208a36f468c5c)